clients/web/test/spec/dataSpec.js

```diff
@@ -861,5 +861,13 @@ describe('Test FileModel static upload functions', function () {
         });
     });
 
-    it('logout from test account', girderTest.logout('logout from test account'));
+    it('logout from test account', function () {
```

Comment:
I can help run down the timing issues on the web client tests.  Superficially, this change looks like it is the cause of intermittent web_client_data* failures, because there isn't an appropriate wait after the click (and possibly before it).

The web_client_group test failure is unrelated to this PR, as I've seen it occasionally anyway.